### ISSUE TITLE
Changes to Dockerfile in version 3.3.0 break connectivity in ECS/EC2 docker environment

### ISSUE THREAD

**Issue by ekinsokmen** [aws/aws-xray-daemon#135]
AWS x-ray daemon is part of our CloudFormation based ECS deployment using the following config:
```
        - Image: amazon/aws-xray-daemon
          Name: xray
          PortMappings:
            - ContainerPort: 2000
              HostPort: 0
              Protocol: udp
```
The change in version >= 3.3.0 did break the connectivity between our service and the daemon because the config option `-t 0.0.0.0:2000`  is removed from the Dockerfile. 

We are getting `Encountered error polling GetSamplingRules ... Connection refused` error from the agent while getting the sampling rules.

**Comment by awssandra** [comment]
Hey ekinsokmen,

Apologies for the issue and thanks for the alert, we'll be pushing out a fix for the images ASAP.

**Comment by willarmiros** [comment]
@ekinsokmen we hope to have this fixed soon. For an immediate workaround, we recommend using the `3.2.0` tag from Dockerhub which has the desired config options.

**Comment by ekinsokmen** [comment]
> @ekinsokmen we hope to have this fixed soon. For an immediate workaround, we recommend using the `3.2.0` tag from Dockerhub which has the desired config options.

Yes, we did that already. thx.

**Comment by awssandra** [comment]
Hi ekinsokmen,

New tag "alpha" is up - can you confirm that this image is working as intended? We are moving forward with the release in the meantime.

**Comment by ekinsokmen** [comment]
Hi, "alpha" seems to be working as expected without error logs. 👍

**Comment by willarmiros** [comment]
@ekinsokmen We have released the fix to bind the daemon to listen to `0.0.0.0:2000` by default and expose port 2000 to TCP and UDP traffic in the `3.3.2`, `3.x`, and `latest` tags, as was the behavior in 3.2.0. You should be able to switch back to one of those tags without issue.
